src/main/java/io/confluent/examples/streams/WordCountLambdaExample.java

```diff
@@ -111,8 +111,48 @@
  */
 public class WordCountLambdaExample {
 
+  static final String inputTopic = "inputTopic";
```

Comment:
These topic names need to match the names of the directions in the javadoc
`inputTopic`= `streams-plaintext-input`
`outputTopic` = `streams-wordcount-output`

Reply:
Good catch. Fixed.

Reply:
Problem was inherited from WordCountLambdaIntegrationTest.java